src/pages/patientView/PatientViewPage.tsx

```diff
@@ -658,6 +679,87 @@ export default class PatientViewPage extends React.Component<
                                         uniqueSampleKey={sample.id}
                                     />
                                 )}
+
+                            {this.patientViewPageStore
```

Comment:
Contents of these two spans and the render conditions are pretty similar to each other, it might be better to have a small customizable functional component for it.